include/sw/universal/number/unum2/_unum2_impl.hpp

```diff
@@ -0,0 +1,320 @@
+#pragma once
+// unum2.hpp: definition of the flexible configuration universal number system
+//
+// Copyright (C) 2017-2021 Stillwater Supercomputing, Inc.
```

Comment:
need the more modern copyright header that is being used throughout the rest of the library:

// Copyright (C) 2017 Stillwater Supercomputing, Inc.
// SPDX-License-Identifier: MIT